### PR TITLE
Improve npm_and_yarn handler

### PR DESCRIPTION
**Pull request by LeoColomb**
Same as #934, but for npm & yarn.

This PR is basically only "meta" changes to reduce yarn load by using yarn workspaces. Nothing huge.

### CODE REVIEW

**Comment by greysteil** [comment]
Looks good to me - thanks @LeoColomb!

@feelepxyz - any comments?

File: npm_and_yarn/helpers/build [hunk 1]
```diff
@@ -10,5 +10,6 @@ fi
 
 helpers_dir="$(dirname "${BASH_SOURCE[0]}")"
 cd "$helpers_dir"
-cd npm && yarn install && cd -
-cd yarn && yarn install && cd -
+yarn install
+yarn lint
+# yarn test
```
Comment (feelepxyz) [inline npm_and_yarn/helpers/build]: We run lint and test in circle so could remove both from here and just do an install.

**Comment by LeoColomb** [comment]
> Do you mind squashing your commits into one?

I definitely can, but I admit I prefer you use the squash option when merging instead. Is it possible for you? Let me know.

**Comment by feelepxyz** [comment]
@LeoColomb yes nice one! Will squash this

**Pull request merged**